substrate/frame/revive/src/storage.rs

```diff
@@ -75,6 +75,8 @@ pub struct ContractInfo<T: Config> {
 	/// to the map can not be removed from the chain state and can be safely used for delegate
 	/// calls.
 	delegate_dependencies: DelegateDependencyMap<T>,
+	/// The size of the immutable data of this contract.
+	pub immutable_bytes: u32,
```

Comment:
Please no new pub fields here. I want to restrict modification outside of this module  if possible. The existing pub fields are just legacy could that need to be converted.

Reply:
Makes sense, changed in favor of getters